src/connection/xmpp/ConnectHelper.ts

```diff
@@ -19,9 +19,55 @@ export function login() {
    }
 }
 
-function loginWithPassword(url: string, jid: string, password: string): Promise<{}> {
+export function websocket(connection,url:string) {
+
+    connection.resume();
+    connection.flush();
+    SM.changeState(SM.STATE.READY);
+    connection.resume();
+    connection.flush();
```

Comment:
Why was this required?

Reply:
the connection was not sending packets anymore without it... maybe problem in underlying strophe connection if change from http to websocket was done